### DISCUSSION TITLE
How to reduce the dimension of the vectors?

### DISCUSSION THREAD

**Discussion by andy263** [milvus-io/milvus#16056] [Q&A and General discussion]
tensorflow encoder in bootcamp proj outputs a 2048 dimensional vectors, towhee encoder outputs 1000 dimensions. is it possible to reduce the dimension to 512 or lower?  What is the best practice vector dimension for reverse image search?

Thanks.

**Comment by yhmo** [comment]
There was a discussion: https://github.com/milvus-io/milvus/discussions/15554
